src/ui/mainwindow.cc

```diff
@@ -3165,8 +3165,7 @@ void MainWindow::iconSizeActionTriggered( QAction * /*action*/ )
   //adjust the font size as well
   auto font = translateBox->translateLine()->font();
   font.setWeight( QFont::Normal );
-  //arbitrary value to make it look good
-  font.setPixelSize( extent * 0.8 );
+  font.setPointSize( QApplication::font().pointSize() );
   //  translateBox->completerWidget()->setFont( font );
   //only set the font in toolbar
   translateBox->translateLine()->setFont( font );
```

Comment:
Why not just delete this?

BTW, this code doesn't adjust the groupCombox's font size.

Reply:
users can use preference->font size now.

Reply:
> this code doesn't adjust the groupCombox's font size.

intended ,it will make macos looks ugly.